meta.go

```diff
@@ -277,9 +279,20 @@ func ReadMeta(r io.Reader) (meta *Meta, err error) {
 	meta.Series = parseStringList(m["series"])
 	meta.Storage = parseStorage(m["storage"])
 	meta.PayloadClasses = parsePayloadClasses(m["payloads"])
+
+	if v, ok := m["min-juju-version"].(string); ok {
```

Comment:
surely missing is fine, and valid is fine, but I think invalid data should be considered an error